cmd/ttn-lw-stack/commands/as_db.go

```diff
@@ -82,10 +82,10 @@ var (
 						}
 						var any bool
 						for _, sess := range []*ttnpb.Session{dev.Session, dev.PendingSession} {
-							if sess == nil || sess.StartedAt.IsZero() {
+							if sess == nil || sess.StartedAt == nil {
 								continue
 							}
-							sess.StartedAt = time.Time{}
+							sess.StartedAt = nil
```

Comment:
Does this result in another migration?

The old version reset `started_at` to `time.Time{}`, this version resets it to `nil`, which I think is a different binary value (field is set to default Go value vs field is nil).

Reply:
This is indeed an unset binary compared to the old zero value. We can up the schema version in the next minor. This will also cause the field to no longer be part of the JSON marshaling.

Reply:
So this has to wait until 3.17.0, because we're changing database schema?

Reply:
No, but in 3.17.0 we can basically rerun this to clean the zeroes. But no need to stall this.